### PR TITLE
Feat/test utils

### PR DESCRIPTION
**Pull request by scottPlusPlus**
Adding TestHelper to (hopefully) make testing more convenient.

So far only applied to createElection and editElection tests.

land atop feat/mock-db-serialization

### CODE REVIEW

File: backend/src/Services/Logging/TestLoggerImpl.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+export class TestLoggerImpl {
+
+    logs:string[] = [];
+
+    constructor() {
+    }
+
+    debug(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
+        this.log(context, "", message, ...optionalParams);
```
Comment (ArendPeter) [inline backend/src/Services/Logging/TestLoggerImpl.ts:14]: Should debug and info also have the "DEBUG" and "INFO" prefixes?

File: backend/src/test/createElection.test.ts [hunk 2]
```diff
@@ -1,71 +1,72 @@
...
 
 afterEach(() => {
-    jest.clearAllMocks();
+  jest.clearAllMocks();
+  th.afterEach();
 });
 
 describe("Create Election", () => {
+  describe("Good Election data provided", () => {
+    var electionRes: Election;
```
Comment (ArendPeter) [inline backend/src/test/createElection.test.ts:23]: Nice! This looks a lot cleaner

**Review by ArendPeter — APPROVED** [review]
Looks good!

File: backend/src/test/createElection.test.ts (hunk 2 above)
Comment (mikefranze) [reply backend/src/test/createElection.test.ts:23]: Agreed!

File: backend/src/Services/Logging/TestLoggerImpl.ts (hunk 1 above)
Comment (scottPlusPlus) [reply backend/src/Services/Logging/TestLoggerImpl.ts:14]: I just copied what's in the normal logger.

My general thought was that the debug and info logs would be the "normal" ones.  We would want warn and errors to stand out more.

However, if we're using new relic to filter things anyway, may make sense to add the prefix?

**Pull request merged**